sdk/communication/communication-phone-numbers/swagger/README.md

```diff
@@ -11,8 +11,8 @@ package-version: 1.0.1
 generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-tag: package-phonenumber-2021-03-07
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/896d05e37dbb00712726620b8d679cc3c3be09fb/specification/communication/data-plane/PhoneNumbers/readme.md
+tag: package-phonenumber-2022-01-11-preview2
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/communication/data-plane/PhoneNumbers/readme.md
```

Comment:
I would recommend pinning this to a specific git commit sha if possible - unless there's some reason I am not thinking of not to?

Reply:
@automagically here's the link pinned to the specific commit sha: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/02fce64677f78021ba70d69bb8bd0d916d653927/specification/communication/data-plane/PhoneNumbers/readme.md

Reply:
@maorleger I agree, makes more sense to pin it to a sha, just a mistake.  
@lucasrsant Thanks for the correct link!